.yamato/upm-ci.yml

```diff
@@ -59,6 +59,34 @@ bizarro_standalone_platforms:
   #  runtime: stadia
 
 ---
+docs-validate:
```

Comment:
I would add this as a dependency on Test_Trigger ? (the one triggered by every PR)

Reply:
Good catch. Just added it to test_trigger:)